android/widgets/src/main/java/org/nativescript/widgets/Async.java

```diff
@@ -296,10 +297,20 @@ public void addHeaders(HttpURLConnection connection)
 				{
 					return;
 				}
+				boolean hasAcceptHeader = false;
 
 				for (KeyValuePair pair : this.headers)
 				{
-					connection.addRequestProperty(pair.key.toString(), pair.value.toString());
+					String key = pair.key.toString();
+					connection.addRequestProperty(key, pair.value.toString());
+					if (key.toLowerCase().contentEquals("Accept-Encoding")) {
```

Comment:
Should be `if (key.toLowerCase().contentEquals("accept-encoding"))` ?

Reply:
Bah, what an idiot I am...   :grinning:      Yes I added the toLowerCase because I found out it could be any case sensitivity; and then forgot to fix the hard coded string to match...  LOL.   I'll toss another patch in...